interface-definitions/include/version/static-routing-version.xml.i

```diff
@@ -0,0 +1,3 @@
+<!-- include start from include/version/static-routing-version.xml.i -->
+<syntaxVersion component='static-routing' version='1'></syntaxVersion>
```

Comment:
I wonder if we should introduce a new component here or rely on the good old fashioned `quagga-version` here.